projects/packages/account-protection/src/class-password-strength-meter.php

```diff
@@ -133,14 +135,16 @@ public function enqueue_jetpack_password_strength_meter_reset_script(): void {
 	 * @return void
 	 */
 	public function localize_jetpack_data( bool $user_specific = false ): void {
+		$jetpack_logo = new Jetpack_Logo();
+
 		wp_localize_script(
 			'jetpack-password-strength-meter',
 			'jetpackData',
 			array(
 				'ajaxurl'                => admin_url( 'admin-ajax.php' ),
 				'nonce'                  => wp_create_nonce( 'validate_password_nonce' ),
 				'userSpecific'           => $user_specific,
-				'logo'                   => plugin_dir_url( __FILE__ ) . 'assets/jetpack-logo.svg',
+				'logo'                   => htmlspecialchars( $jetpack_logo->get_jp_emblem( true ), ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401 ),
```

Comment:
```suggestion
				'logo'                   => htmlspecialchars( $jetpack_logo->get_jp_emblem( true ) ),
```
It is my impression that [this is the default value](https://www.php.net/manual/en/function.htmlspecialchars.php) for this param, can it be removed?

Reply:
I've added this only in response to a failing PR check that requested it - I'll test again with it removed but if the checks fail again, I'll revert and merge 👍🏻

Reply:
Yep, PHP compatibility for dev branch PHP 8.0 fails without that explicit declaration:
```
The default value of the $flags parameter for htmlspecialchars() was changed from ENT_COMPAT to ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401 in PHP 8.1. For cross-version compatibility, the $flags parameter should be explicitly set. (PHPCompatibility.ParameterValues.NewHTMLEntitiesFlagsDefault.NotSet)
```

Reverting and will merge as-is!